aspnetcore/security/authentication/azure-active-directory/index.md

```diff
@@ -17,3 +17,11 @@ uid: security/authentication/azure-active-directory/index
 * [Calling a Web API in an ASP.NET Core Web Application Using Azure AD](https://azure.microsoft.com/documentation/samples/active-directory-dotnet-webapp-webapi-openidconnect-aspnetcore/)
```

Comment:
@Rick-Anderson, I wonder if you don't want to have:
## Azure AD V1 samples
Then the first 2 samples that had initially

## Azure AD B2C sample
The 4th sample

## Azure AD V2 samples
What you just added

Do you want me to send a PR? (but that would not be before Monday)à

Reply:
> Do you want me to send a PR? 

Next week would be great.

Reply:
ok will do.